src/content/dvt/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Distributed Validator Technology
+description: Distributed Validator Technology enables the distributed operation of an Ethereum validator by multiple parties.
+lang: en
+---
+
+# Distributed Validator Technology {#what-is-dvt}
+
+
+
+**Distributed Validator Technology (DVT)** splits the private key used to secure a validator across many computers organized into a "cluster". The benefit of this is that it makes it very difficult for attackers to gain access to the key, because it is not stored in full on any single machine. It also allows for some nodes to go offline, as the necessary signing can be done by a subset of the machines in each cluster. This reduces single points of failure from the network and makes the whole validator set more robust. 
+
+
+Distributing a new or existing validator is quite complex in practice, but using a DVT solution like ssv.network as a layer to split the validator private key into multiple key shares makes the technology more accessible. Regardless of the [DVT-implementation](<https://ssv.network/>) stakers can have multiple nodes each operating a key share in a cluster, allowing them to ensure the uptime of their validators.
+
+## How does DVT work? {#how-does-dvt-work}
+
+A DVT solution contains the following components :
+
+![A Diagram showing how a single validator key is split into key shares and distributed to multiple nodes with varying components.](./dvt-cluster.png)
```

Comment:
yea this is a bit awkward - we might be able to get some designers involved to help. Basically we would want to either recreate the entire image as a react component that can conditionally render the labels depending on the site language, or replace the text in the image with symbols that can be explained in text (so that the entire page text can be translated into many languages without leaving language-specific labels in the image. 

As there are only two relevant labels (key share and validator key) the latter option seems fine - the other text elements are client names which don't need translation (@wackerow agree?)

I don't think this necessarily has to be a blocker, but it will need to be done asap after the page goes live.

Reply:
we tried to find an original source for this but the current .png file  is the only we have unfortunately :)